### DISCUSSION TITLE
Q&A 应该与 Feature Request 更换位置或者删除 Feature Request，感觉 Discussion 区有点乱

### DISCUSSION THREAD

**Discussion by warmthsea** [element-plus/element-plus#16877] [Ideas]
更改选择排序或者删除 Feature Request

![image](https://github.com/element-plus/element-plus/assets/45450994/39f6d3b9-b73e-4a6f-aeda-fa66f6143970)

![image](https://github.com/element-plus/element-plus/assets/45450994/c8662369-c3e3-4980-bc07-37238e64082d)

**Comment by xxholly32** [comment]
好建议无脑抄vue的好了

![image](https://github.com/element-plus/element-plus/assets/6063358/88cc422b-e061-4fa9-92cb-df6d8078ef94)
